satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -333,7 +333,7 @@ model bucket_usage (
     unique rollup_end_time bucket_id
 
     field  id                 blob
-    field  bucket_id          blob      //--TODO: add foreign key constraint--//
+    field  bucket_id          text      //--TODO: add foreign key constraint--//
```

Comment:
this should be a blob, text implies valid utf8

Reply:
the bucket id will be a string in the form "projectname/bucketname".  should it still be a blob?

Reply:
Yes. I think we should have the project name in a separate field. Unless there's a reason we cannot keep them in the same field, I guess.

Reply:
Currently there is no concept of a proper bucketID that is a blob. There is only this bucket name string.  I can add a separate field for the bucket_id string, I guess I will call it `bucket_name` instead, but for now that would be the bucket_id blob field will always be empty.  Does that sound right?

Reply:
Note, the correct translation of `string` in Go for databases usually is `[]byte`, `blob` or `bytes` because Go strings are not guaranteed to be utf-8. There must be other checks in place for them to be considered utf-8.

Reply:
So keep the bucket_id blob field.  Add a project_name string field, and add a bucket_name string field.  Is that what you mean?

Reply:
I would do:
```
field project_id blob
field bucket_id blob
```

Reply:
oh ok i didn't know string could mean `blob` in database

Reply:
blob is effectively a `[]byte` and go strings are `[]byte` with some sugar.